config

```diff
@@ -0,0 +1,66 @@
+####################
```

Comment:
I don't see a .gitignore entry for this file. As we discussed on the phone, we really don't want someone accidentally changing this in a PR, so a .gitignore entry will help with that.